packages/esm-appointments-app/src/appointment-forms/appointments-form.component.tsx

```diff
@@ -53,7 +53,7 @@ interface AppointmentFormProps {
   patientUuid?: string;
   context: string;
 }
-const AppointmentForm: React.FC<AppointmentFormProps> = ({ appointment = {}, patientUuid, context }) => {
+const AppointmentForm: React.FC<AppointmentFormProps> = ({ appointment, patientUuid, context }) => {
```

Comment:
This change isn't going to break anything is it?

Reply:
It is not I confirmed it.